docs/source/models.rst

```diff
@@ -3655,23 +3655,113 @@ Evaluating with Extra Metrics
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 If the default set of metrics is insufficient, you can supply ``extra_metrics`` and ``custom_artifacts``
-to :py:func:`mlflow.evaluate()` to produce custom metrics and artifacts for the model(s) that you're evaluating.
+to :py:func:`mlflow.evaluate()` to produce extra metrics and artifacts for the model(s) that you're evaluating.
+
+To define an extra metric, you should define an ``eval_fn`` function that takes in ``predictions``, ``targets``,
```

Comment:
Can we mention what data type ``predictions`` and ``targets`` contains, and what is the content of them? For example, if the `data` is a pandas dataframe, then ``predictions`` and ``targets`` are pandas series, and the content is specified by the corresponding parameters in ``mlflow.evaluate()``. I'm not sure if predictions and targets could be other data types. Maybe you can confirm with @annzhang-db or look into the code.